docs/how_to/create_a_parameterized_circuit.rst

```diff
@@ -0,0 +1,114 @@
+##############################
+Create a parameterized circuit
```

Comment:
In the other titles about circuits they qualify this as a `quantum circuit`. This is the only title where its not explicitly stated as `quantum`.

Reply:
Added "quantum" to the title in [afa30d9](https://github.com/Qiskit/qiskit-terra/pull/8624/commits/afa30d97fa0d513420e332755d660eb6d18e2a33)